src/arrayMethodSort.js

```diff
@@ -4,8 +4,56 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function(compareFunction = .......) {
```
add the default value for that function

Reply:
add default value